man/ff-deps.1.md

```diff
@@ -0,0 +1,135 @@
+% FF-DEPS(1)
+% Daniel de Kok, Sebastian Pütz
+% Apr 6, 2019
+
+NAME
+====
+
+**ff-deps** -- train dependency-based word embeddings with subword representations
+
+SYNOPSIS
+========
+
+**ff-deps** [*options*] *corpus* *output*
+
+DESCRIPTION
+===========
+
+The **ff-deps** trains dependency based word embeddings (Levy and Goldberg, 
+2014) using data from a *corpus* in CONLLX format. The corpus contains 
+sentences seperated by empty lines. Each sentence needs to be annotated with
+a dependency parse. After training, the embeddings are written to 
+*output* in the finalfusion format.
+
+OPTIONS
+=======
+
+`--buckets` *EXP*
+
+:   The bucket exponent. finalfrontier uses 2^*EXP* buckets to store subword
+    representations. Each subword representation (n-gram) is hashed and
+    mapped to a bucket based on this hash. Using more buckets will result
+    in fewer bucket collisions between subword representations at the cost
+    of memory use. The default bucket exponent is *21* (approximately 2
+    million buckets).
+
+`--context_discard` *THRESHOLD*
+
+:   The context discard threshold influences how often frequent contexts are
+    discarded during training. The default context discard threshold
+    is *1e-4*.
+    
+`--context_mincount` *FREQ*
+
+:   The minimum count controls discarding of infrequent contexts. Contexts 
+    occuring fewer than *FREQ* times are not considered during training. 
+    The default minimum count is 5.
+    
+`--dims` *DIMS*
+
+:   The dimensionality of the trained word embeddings. The default
+    dimensionality is 100.
+
+`--dependency_depth` *DEPTH*
+
+:   Dependency contexts up to *DEPTH* distance from the focus word in the
+    dependency graph will be used to learn the representation of the focus
+    word. The default depth is *1*.
+
+`--discard` *THRESHOLD*
+
+:   The discard threshold influences how often frequent words are discarded
+    from training. The default discard threshold is *1e-4*.
+
+`--epochs` *N*
+
+:   The number of training epochs. The number of necessary training epochs
+    typically decreases with the corpus size. The default number of epochs
+    is *5*.
+
+`--lr` *LEARNING_RATE*
+
+:   The learning rate determines what fraction of a gradient is used for
+    parameter updates. The default initial learning rate is *0.05*, the
+    learning rate decreases monotonically during training.
+
+`--maxn` *LEN*
+
+:   The maximum n-gram length for subword representations. Default: 6
+
+`--mincount` *FREQ*
+
+:   The minimum count controls discarding of infrequent. Words occuring
```

Comment:
I guess in this manpage (as opposed to `ff-train`) it might be better to say 'Focus words occurring'... to contrast with context words?